example/GetEnrollmentReportLiveTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * A live Enrollment Report test.
+ *
+ * This test expects the SMARTERU_ACCOUNT_KEY and SMARTERU_USER_KEY environment
+ * variables to be set. It will query the SMARTERU API for a list of all users
+ * in the SANDBOX group that are active and output their progress in the
+ * courses they are enrolled in, in CSV format.
+ *
+ * @copyright   $year$ Core Business Solutions
+ * @license     MIT
+ * @version     $version$
+ */
+ 
+declare(strict_types=1);
+
+namespace CBS\SmarterU\Tests\Usability;
+
+require_once(__DIR__ . '/../vendor/autoload.php');
+
+use CBS\SmarterU\Client;
+use CBS\SmarterU\Exceptions\SmarterUException;
+use CBS\SmarterU\Queries\GetLearnerReportQuery;
+
+$accountKey = getenv('SMARTERU_ACCOUNT_KEY') ?? 'No Account Key Provided';
+$userKey = getenv('SMARTERU_USER_KEY') ?? 'No User Key Provided';
```

Comment:
I think we ought to print a help message here if the values are not set rather than firing off a request that we know will fail.